tasks/kots_install.rb

```diff
@@ -64,6 +64,11 @@ def generate_config(license, hostname, password)
     config = base_config(appname, hostname)
     spec_values = config['spec']['values']
     spec_values.merge!(root_account_config(appname, password))
+    if appname == 'connect'
```

Comment:
Doesn't this need to also check the license for `data_entitlement`?  Or is the beta agreement needed for either?

Reply:
I was kind of leaving `data_entitlement` for later, but I could work on adding it.

Reply:
We can circle back to that later if we just need to unblock CI for https://github.com/puppetlabs/holodeck-manifests/pull/1287 now.  I'm cool with that.